genai-function-calling/vercel-ai/index.js

```diff
@@ -52,7 +52,12 @@ const tools = {get_latest_elasticsearch_version: getLatestElasticsearchVersion};
  */
 async function runAgent(tools) {
     const {text} = await generateText({
-        model: openai(model),
+        // If using qwen3, remove the reasoning tags from output. GPT models will not have
```

Comment:
we can say "format rewards" I think as digging around it is present in deepseek-r1 and families like it, and even intentionally tested! https://github.com/huggingface/open-r1/blob/main/tests/test_rewards.py#L429

Reply:
Not sure I got the format reward aspect but reworded to be more generic